torch/_dynamo/variables/torch.py

```diff
@@ -314,6 +314,9 @@ def call_function(
             return NullContextVariable(**options)
         elif self.value is torch.autograd._profiler_enabled:
             unimplemented("torch.autograd._profiler_enabled not supported yet")
+        elif self.value is torch.cuda.stream:
```

Comment:
How do we add stream from other backends, do we need to add a similar change here?

Reply:
Yes.